.drone.star

```diff
@@ -224,10 +224,88 @@ config = {
 			'extraEnvironment': {
 				'S3_TYPE': 'scality',
 			},
+			'scalityS3': True,
```

Comment:
This should have always been set. It ends up causing `~@skip_on_objectstore` to be added to the filter tags. That is essential if we run all API scenarios - we want to skip those tagged `skip_on_objectstore`. When running just the smoke tests, it happens to make no difference. None of the smoke tests have `skip_on_objectstore`.